recipes/lzma_sdk/9.20/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,10 @@
+from conans import ConanFile, tools
```

Comment:
The ``test_v1_package`` is no longer necessary (it doesn't harm to be there, but you can skip it if you want)